### PR TITLE
Add favicons to application

### PR DESCRIPTION
**Pull request by asolvang**
Adds favicons to application. 

Used a gem for it, that way we don't (hopefully) have to worry about how it looks in different browser, media devices, ect. 

Please test in different browser, different operating systems.
Also try to "pin" the app, or create a shortcut to it to see how the icon turns out.

### CODE REVIEW

**Comment by asolvang** [comment]
[Favicon](https://trello.com/c/Uo8PLBT9/236-favicon)

File: Gemfile [hunk 1]
```diff
@@ -92,3 +92,7 @@ gem "simple_form"
 gem "simple_form-tailwind"
 gem "tailwindcss-rails-webpacker", "~> 0.1.2"
 gem "validate_url"
+
+group :development do
+  gem "rails_real_favicon"
+end
```
Comment (mathias234) [inline Gemfile:98]: Trenger ikke en egen :development group her

**Review by mathias234 — APPROVED** [review]
LGTM! Har testet det på flere steder og ser ut til å fungere

**Review by DanielJackson-Oslo — APPROVED** [review]
LGTM!

**Pull request merged**